xblock/internal.py

```diff
@@ -28,42 +27,3 @@ def __get__(self, instance, owner):
 
 
 class_lazy = LazyClassProperty  # pylint: disable=invalid-name
-
-
-class NamedAttributesMetaclass(type):
```

Comment:
NamedAttributesMetaclass and Nameable have been de-generalized into xblock.core._AutoNamedFieldsMetaclass and xblock.fields.Field, respectively.